src/script/common/c_internal.cpp

```diff
@@ -157,19 +157,20 @@ static void script_log(lua_State *L, const std::string &message,
 
 void log_deprecated(lua_State *L, const std::string &message, int stack_depth)
 {
-	static bool configured = false;
-	static bool do_log     = false;
-	static bool do_error   = false;
+	static thread_local bool configured = false;
```

Comment:
either all of these or none should be thread local

Reply:
fixed